src/gui/previewselectdialog.h

```diff
@@ -40,7 +40,7 @@ class QStandardItemModel;
 
 class PreviewListDelegate;
 
-class PreviewSelect: public QDialog, private Ui::preview
+class PreviewSelectDialog: public QDialog, private Ui::preview
```

Comment:
Could you fix coding style (spaces around colon `...PreviewSelectDialog : public...`)

Reply:
done